src/sql/workbench/parts/notebook/browser/models/cell.ts

```diff
@@ -603,7 +606,7 @@ export class CellModel implements ICellModel {
 		this._source = this.getMultilineSource(cell.source);
 		this._metadata = cell.metadata || {};
 
-		if (this._metadata.tags && this._metadata.tags.some(x => x === HideInputTag)) {
+		if (this._metadata.tags && this._metadata.tags.some(x => x === HideInputTag) && this._cellType === CellTypes.Code) {
```

Comment:
Isn't this unnecessary? isCollapsed is just going to ignore it anyways if the cell type isn't code.